src/features/metadata-instance-editor/Instance.js

```diff
@@ -102,6 +102,7 @@ class Instance extends React.PureComponent<Props, State> {
     static defaultProps = {
         data: {},
         isDirty: false,
+        isCascadingPolicyApplicable: false,
```

Comment:
sorry do we even need to set this explicitly? shouldn't all usages be inherited from the default set in `Instances`?
https://github.com/box/box-ui-elements/blob/9ab6d78b09214d5928add084e5cd72ab8d8631b4/src/features/metadata-instance-editor/Instances.js#L22-L23

Reply:
You're right but I suppose that it doesn't hurt to set it here as well?